content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -55,45 +55,47 @@ The example `rippled-example.cfg` file has the `type` field in the `[node_db]` s
 
 [RocksDB](https://rocksdb.org/docs/getting-started.html) is an embeddable persistent key-value store that is optimized for rotational disks.
 
-RocksDB requires approximately one-third less disk [storage](#storage) than NuDB and provides better I/O latency. However, the better I/O latency comes as result of the large amount of RAM RocksDB requires to store data indexes.
+RocksDB requires approximately one-third less [disk storage](#disk-space) than NuDB and provides better I/O latency. However, the better I/O latency comes as result of the large amount of RAM RocksDB requires to store data indexes.
 
-Validators should be configured to use RocksDB and to store no more than about 300,000 ledgers (approximately two weeks' worth of [historical data](#historical-data)) in the ledger store.
+Validators should be configured to use RocksDB and to store no more than about 300,000 ledgers (approximately two weeks' worth of [historical data](#disk-space)) in the ledger store.
 
 RocksDB has performance-related configuration options that you can set in `rippled.cfg` to achieve maximum transaction processing throughput. Here is the recommended configuration for a `rippled` server using RocksDB:
 
 ```
 [node_db]
 type=RocksDB
-path={path_to_ledger_store}
+path=/var/lib/rippled/db/rocksdb
 open_files=512
 filter_bits=12
 cache_mb=512
 file_size_mb=64
 file_size_mult=2
+online_delete=2000
+advisory_delete=0
 ```
 
+(Adjust the `path` to the directory where you want to keep the ledger store on disk. Adjust the `online_delete` and `advisory_delete` settings as desired for your configuration.)
+
 #### More About Using NuDb
 
 [NuDB](https://github.com/vinniefalco/nudb#introduction) is an append-only key-value store that is optimized for SSD drives.
 
-NuDB has nearly constant performance and memory footprints regardless of the amount of data being [stored](#storage). NuDB _requires_ a solid-state drive, but uses much less RAM than RocksDB to access a large database.
+NuDB has nearly constant performance and memory footprints regardless of the [amount of data being stored](#disk-space). NuDB _requires_ a solid-state drive, but uses much less RAM than RocksDB to access a large database.
```

Comment:
SSD's are recommended for all configurations, not just nudb.

Reply:
Yes, that is clarified a couple other places in this document, in particular lines 44-50 which lead into this section.